src/validations/businessCaseSchema.ts

```diff
@@ -2,6 +2,167 @@ import * as Yup from 'yup';
 
 //
 const phoneNumberRegex = /( *-*[0-9] *?){10,}/;
+
+const estimatedLifecycleCostSchema = Yup.object().shape({
+  year1: Yup.object()
+    .test(
```

Comment:
This validation makes sure each year has AT LEAST one checkbox (development, O&M, or other) are selected.

Reply:
I wasn't able to get this validation to fail when I didn't select a checkbox.

Reply:
🤔 Can you make sure the associated intake has status `BIZ_CASE_FINAL_NEEDED`? We're only running validations on final business case submissions (not draft).

Reply:
@christopherhuii oh, that was probably the issue! I was in draft.